client/app/page.tsx

```diff
@@ -22,80 +41,200 @@ export default function HomeScreen() {
-          <button style={styles.item} onClick={() => triggerKeyPress(serverIP, "brightness_up")}>
-            <Image width="25" height="25" alt="brightness up" src={`/images/brightness-up.png`} />
+        <div style={styles.row}>
+          <button
+            style={styles.item}
+            onClick={() => triggerKeyPress(serverIP, 'previous_track')}
+          >
+            <Image
+              width='25'
+              height='25'
+              alt='previous track'
+              src={`/images/light/track-prev.png`}
+            />
           </button>
-          <button style={styles.item} onClick={() => triggerKeyPress(serverIP, "brightness_down")}>
-            <Image width="25" height="25" alt="brightness down" src={`/images/brightness-down.png`} />
+          <button
+            style={styles.item}
+            onClick={() => triggerKeyPress(serverIP, 'next_track')}
+          >
+            <Image
+              width='25'
+              height='25'
+              alt='next track'
+              src={`/images/light/track-next.png`}
+            />
           </button>
         </div>
 
-        <button style={styles.item} onClick={() => reScan(i, incrRescan, setServerIP)}>
-          <Image width="25" height="25" alt="rescan" src={`/images/rescan.png`} />
-        </button>
-
-        <div style={styles.column}>
-          <button style={styles.item} onClick={() => adjustVolume(serverIP, "volume_up", setVolume)}>
-            <Image width="25" height="25" alt="volume up" src={`/images/volume-up.png`} />
+        <div style={styles.row}>
+          <button
+            style={styles.item}
+            onClick={() => triggerKeyPress(serverIP, 'left_arrow')}
+          >
+            <Image
+              width='25'
+              height='25'
+              alt='left arrow key'
+              src={`/images/light/arrow-left.png`}
+            />
+          </button>
+          <button
+            style={styles.item}
+            onClick={() => triggerKeyPress(serverIP, 'play_pause')}
+          >
+            <Image
+              width='25'
+              height='25'
+              alt='play/pause'
+              src={`/images/light/play.png`}
+            />
           </button>
-          <button style={styles.item} onClick={() => adjustVolume(serverIP, "volume_down", setVolume)}>
-            <Image width="25" height="25" alt="volume down" src={`/images/volume-down.png`} />
+          <button
+            style={styles.item}
+            onClick={() => triggerKeyPress(serverIP, 'right_arrow')}
+          >
+            <Image
+              width='25'
+              height='25'
+              alt='right arrow key'
+              src={`/images/light/arrow-right.png`}
+            />
           </button>
         </div>
 
-      </div>
+        <div style={styles.buttonRow}>
+          <BrightnessControls serverIP={serverIP} />
+
+          <div style={styles.column}>
+            <div
+              className={poppins.className}
```

Comment:
is adding poppins again here necessary if we add it to top level component? (maybe throw it on layout.tsx?)